youtubedl/pyqt.py

```diff
@@ -51,6 +51,10 @@ def enterURL(self):
         """
 
         link = self.lineEditURL.text()
+        #Checks if a youtube url is input. otherwise looks like it freezes and gives no output.
+        correct_url = "www.youtube.com/watch?"
+        if correct_url not in link:
+                self.showPopUp("Please input correct url.\nFor example: 'https://www.youtube.com/watch?v=9bZkp7q19f0'")
```

Comment:
Just an observation. I think the error message should be clear in that for now we only support YouTube downloads. Maybe "Please input a YouTube URL. For example 'https://www.youtube.com/watch?v=9bZkp7q19f0'"

Reply:
Probably need to say what the error is. like. "The input url is not a valid YouTube link"

Reply:
Yep. To be honest didnt put much thought into what it should/does say. Yeah one of your ideas are probably more coherent haha. You can change that though? Also an after thought was maybe just create a list of working urls and check if one of those is in url. Thats obviously for other sources and playlists but would also work. could read from .txt to keep the code cleaner and add a .txt with supported urls

Reply:
> Probably need to say what the error is. like. "The input url is not a valid YouTube link"

Even better.

Reply:
barguan, I think Ryan has a better approach for this issue, it tackles other kinds of error and leave room to easy insertion of other situations we stumble upon. I'll take his pull request, ok?